src/main/groovy/net/cockamamy/gradle/javarepl/JavaReplPlugin.groovy

```diff
@@ -48,7 +48,7 @@ class JavaReplPlugin implements Plugin<Project> {
 
             project.javarepl.validate()
 
-            if (JavaVersion.current() != VERSION_1_8 && JavaVersion.current() != VERSION_1_9) {
+            if (JavaVersion.current() == VERSION_1_6 || JavaVersion.current() == VERSION_1_7) {
```

Comment:
If anyone still cares about this, here's a better way to do this check:

```groovy
JavaVersion.current(  ).ordinal(  ) < JavaVersion.VERSION_1_8.ordinal()
```

Reply:
clearly nobody does...